src/dmd/doc.d

```diff
@@ -1844,6 +1856,53 @@ extern (C++) const(char)* skipwhitespace(const(char)* p)
     return p;
 }
 
+/************************************************
+ * Scan past the given characters.
+ */
+size_t skipchars(OutBuffer* buf, size_t i, string chars)
+{
+    const slice = buf.peekSlice();
+    for (; i < slice.length; i++)
+    {
+        bool contains = false;
+        foreach (char c; chars)
+        {
+            if (slice[i] == c)
+            {
+                contains = true;
+                break;
+            }
+        }
+        if (!contains)
+            break;
+    }
+    return i;
+}
+
+/************************************************
+ * Get the indent from one index to another, counting a tab as four spaces.
```

Comment:
Type a text file, tabs are 8. Print a text file, tabs are 8. Counting tabs as 4 doesn't work. I recommend treating tabs as 8, and for anyone who complains, suggest they use spaces. Note that because of this difficulty, no tabs are allowed in dmd source code.

Also, what do you mean by "indent"? The number of columns?